src/Tribe/Aggregator/Record/Queue_Realtime.php

```diff
@@ -196,6 +210,59 @@ public function get_unable_to_continue_processing_data() {
 	 * @return mixed|string|void
 	 */
 	public function get_progress_message_data( $queue, $percentage, $done ) {
+		_deprecated_function( __METHOD__, 'TBD' );
+
+		return wp_json_encode( $this->get_progress_data($queue, $percentage, $done) );
+	}
+
+	/**
+	 * Get the data that is used to construct the current status of the EA progress bar.
+	 *
+	 * @since TBD
+	 *
+	 * @return array<string, mixed> An array with the details of the progress bar.
+	 */
+	private function get_queue_progress_data() {
+		if ( $this->record_id <= 0 ) {
+			return [];
+		}
+
+		// Load the queue
+		/** @var \Tribe__Events__Aggregator__Record__Queue_Interface $queue */
+		$queue = $this->queue ? $this->queue : Tribe__Events__Aggregator__Record__Queue_Processor::build_queue( $this->record_id );
+		// We always need to setup the Current Queue
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 339293a23633d9aada23a257d2ed6464</sub>